getting_started/loadable_modules/lmdk_user_guide.rst

```diff
@@ -7,10 +7,65 @@ What is LMDK
 ************
```

Comment:
Please add a little bit of context in the commit message. 'Howto tutorial' is not really adding much value.

Reply:
Would be easier if build and deploying step worked in CI. However I added richer description I hope that I would be more helpfull.